asteroidsyncserviced/openweathermapparser.cpp

```diff
@@ -0,0 +1,77 @@
+#include "openweathermapparser.h"
+
+OpenWeatherMapParser::OpenWeatherMapParser(QObject *parent) : QObject(parent)
+{
+}
+
+QList<short> OpenWeatherMapParser::getWeatherId(QJsonArray list)
+{
+    QList<short> id;
+    QDateTime timestamp;
+
+    for(int i = 0; i < 5; i++) {
```

Comment:
I recommend using list.size() like in the other methods or you risk accessing undefined values.

Reply:
I think range-based loops should work on QJsonArray https://en.wikipedia.org/wiki/C%2B%2B11#Range-based_for_loop